### PR TITLE
Create function that returns return type of ast node

### PR DESCRIPTION
**Pull request by skurzynowski**

### CODE REVIEW

File: tests/queries/get-return-type.test.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import testObject from '../../src/queries';
+import { transformToAst } from '../helpers';
+
+const testFunctionName = "getReturnType";
```
Comment (Scherlock90) [inline tests/queries/get-return-type.test.ts:4]: You can create a constans.ts or strings.json file in which to put read-only(const) variable

File: tsconfig.json [hunk 2]
```diff
@@ -13,5 +13,6 @@
       "*": ["node_modules/*", "src/types/*"]
     }
   },
-  "include": ["src/**/*", "./example/**/*"]
+  "include": ["src/**/*"],
+  "exclude": ["node_modules","tests/**/*", "**/*.spec.ts"]
```
Comment (Scherlock90) [inline tsconfig.json:17]: please format white space between node _modules and tests

File: src/queries.ts [hunk 3]
```diff
@@ -15,6 +16,26 @@ namespace Queries {
     return nodes;
   };
 
+  export function getReturnType(functionNode: n.FunctionDeclaration): string {
+    let result;
+
+    //handle type reference as array for example
+    if (get(functionNode, 'returnType.typeAnnotation.type') === 'TSTypeReference') {
```
Comment (Scherlock90) [inline src/queries.ts:23]: You can create a constans.ts or strings.json file in which to put read-only(const) variable

File: src/queries.ts [hunk 4]
```diff
@@ -15,6 +16,26 @@ namespace Queries {
...
+      result = get(functionNode, 'returnType.typeAnnotation.typeName.name');
```
Comment (Scherlock90) [inline src/queries.ts:24]: You can create a constans.ts or strings.json file in which to put read-only(const) variable

File: src/queries.ts [hunk 5]
```diff
@@ -15,6 +16,26 @@ namespace Queries {
...
+    }
+
+    //handle annotation like string for example
+    if (!result) {
+      result = get(functionNode, 'returnType.typeAnnotation.type');
```
Comment (Scherlock90) [inline src/queries.ts:29]: You can create a constans.ts or strings.json file in which to put read-only(const) variable

File: tests/queries/get-return-type.test.ts [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+describe(testFunctionName, () => {
+    test(`should return information no return type annotation`, () => {
```
Comment (Scherlock90) [inline tests/queries/get-return-type.test.ts:7]: maybe you should use singlequote not a template string in all tests?

File: src/queries.ts [hunk 7]
```diff
@@ -1,4 +1,5 @@
 import { visit, namedTypes as n } from 'ast-types';
+import { get } from 'lodash';
```
Comment (Scherlock90) [inline src/queries.ts:2]: do we need all lodash, maybe better to used polyfills? the library would have a smaller size

Comment (skurzynowski) [reply src/queries.ts:2]: In the future will be more libraries in this file from loads. So currently is not top prior. :)

File: tests/queries/get-return-type.test.ts (hunk 6 above)
Comment (skurzynowski) [reply tests/queries/get-return-type.test.ts:7]: Tests are out of eslint range and standards.

File: src/queries.ts (hunk 5 above)
Comment (skurzynowski) [reply src/queries.ts:29]: This string currently is used in one place. Ergo is no reason to put this in constant as long as there is no repetition.

File: src/queries.ts (hunk 4 above)
Comment (skurzynowski) [reply src/queries.ts:24]: This string currently is used in one place. Ergo is no reason to put this in constant as long as there is no repetition.

File: src/queries.ts (hunk 3 above)
Comment (skurzynowski) [reply src/queries.ts:23]: This string currently is used in one place. Ergo is no reason to put this in constant as long as there is no repetition.

File: tests/queries/get-return-type.test.ts (hunk 1 above)
Comment (skurzynowski) [reply tests/queries/get-return-type.test.ts:4]: Great idea :) But in tests I need explicit name of function to see and use only here :)

File: src/queries.ts (hunk 7 above)
Comment (skurzynowski) [reply src/queries.ts:2]: 👍

File: tests/queries/get-return-type.test.ts (hunk 6 above)
Comment (skurzynowski) [reply tests/queries/get-return-type.test.ts:7]: 👍

File: src/queries.ts (hunk 5 above)
Comment (Scherlock90) [reply src/queries.ts:29]: line 23 have this same string 'returnType.typeAnnotation.type'. 

The idea of ​​creating a file is that when you create a const in another file, first you can check if it already exists, then there is a small probability of a mistake in writing the same string a second time, and possibly you can use it a second time if necessary. In addition, such a file is a box for read-only variables.

**Pull request merged**